app/javascript/components/pages/Jobs.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import propTypes from 'prop-types';
+import SharedLayout from 'components/layout/SharedLayout';
+import PageTitle from 'components/PageTitle';
+import Button from 'components/Button';
+import { getJobs } from '../../datasources';
+import { postedAtString } from '../../utils';
+
+import 'stylesheets/page';
+import 'stylesheets/jobs';
+
+const Jobs = () => {
+    const [jobs, setJobs] = useState([]);
+
+    useEffect(() => {
+        const fetchData = async () => {
+            const data = await getJobs();
+            setJobs(data);
+        };
+
+        fetchData();
+    }, []);
+
+    const [firstSixJobs, restOfJobs] = useMemo(() => {
+        if (jobs === []) {
+            return [[], []];
+        }
+
+        const firstSix = jobs.slice(0, 6);
+        const rest = jobs.slice(6, jobs.length);
+
+        return [firstSix, rest];
+    }, [jobs]);
```

Comment:
I had an idea in the design to first render the first 6 jobs from the job board, THEN render a banner linking to the sponsorship page (in case somebody's wondering how to get their job up on the job board), THEN the rest of the jobs. So this is the logic that splits up the jobs into the first six and the rest, memoized for optimal performance!